graders/python/requirements.txt

```diff
@@ -15,13 +15,13 @@ openpyxl==3.1.5
 pandas==2.2.3
 Pillow==11.1.0
 PuLP==2.9.0
-Pygments==2.18.0
+Pygments==2.19.1
 pyarrow==18.1.0
 pygraphviz==1.14
 requests==2.32.3
-scikit-image==0.25.0
-scikit-learn==1.6.0
-scipy==1.14.1
+scikit-image==0.25.1
+scikit-learn==1.6.1
+scipy==1.15.1
```

Comment:
This includes some deprecations but nothing removed right away that doesn't already include an error message.